buildSrc/src/main/groovy/DeployableContainerBuilder.groovy

```diff
@@ -109,7 +109,7 @@ abstract class DeployableContainerBuilder extends DefaultTask {
 
     @Input
     final Property<String> baseImageTag =
-            getObjects().property(String).convention('11.0.15-11.56.19')
+            getObjects().property(String).convention('11')
```

Comment:
see @davidcurrie 's comment on a similar PR

Reply:
Yes, I agree - we need to properly "fix" the version like so:
```suggestion
            getObjects().property(String).convention('11.0.16.1-11.58.23')
```